pkg/cmd/status.go

```diff
@@ -109,6 +111,9 @@ var statusCmd = &cobra.Command{
 			if input == "" {
 				return errors.New("Invalid summon tenant name or microservice name")
 			}
+			if utils.SanitizInput(input) {
```

Comment:
now, this time, we are not sanitizing input, we are just checking if input is valid or not.. so you can now rename function to `ValidateInput`

Reply:
Dine